src/components/pages/trialDetail/CardSection.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { colors } from '../../../styles/globalStyles';
+
+import Arrow from '../../common/icons/Arrow.jsx';
+import CardField from './CardField';
+
+const Section = styled.section`
+  width: 70%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: ${colors.white};
+  padding: 1rem;
+  text-align: left;
+  line-height: 1.2rem;
+  margin-bottom: 1rem;
+  box-shadow: ${colors.cardBoxShadow};
+`;
+
+const CenterDev = styled.div`
```

Comment:
This will be a button, it wraps an arrow.
Try using IconButton from Buttons.js. There is an implementation of this on the homepage that may help.

[Some guidance on accessible buttons here](https://developers.google.com/web/tools/lighthouse/audits/button-name)

Reply:
this need to be a link, not a button because we are moving from one route to another